lib/middleman-livereload/reactor.rb

```diff
@@ -40,11 +40,25 @@ def reload_browser(paths = [])
         end
       end
 
+      def websocket_options
+        @websocket_options ||= begin
+          return {} unless @options[:ssl_certificate] || @options[:ssl_private_key]
```

Comment:
Are both required, or just one? If both are required I'd expect `&&` here (or an exception being thrown if only one of them is set).